internal/config/config.go

```diff
@@ -73,11 +75,25 @@ func Read(path string, cfg interface{}) error {
 }
 
 // GetActualValue returns the environment variable value if the val has prefix and suffix "_", otherwise the val will directly return.
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg?style=flat-square)
comments should be updated.

Reply:
okay!

Reply:
fixed about comment thank you for your review